src/UserCollectableAttribute.js

```diff
@@ -12,8 +12,9 @@ const {
   getObjectBasePropName,
   getObjectTypeDefProps,
 } = require('./utils');
+const { UCATemplateValue } = require('./UCATemplateValue');
```

Comment:
You just added but it is doing nothing, it is correct? Wouldn't lint get this?

Reply:
Sorry., I don't understand the comment. it is importing the UCATemplateValue constructor that I removed from the UserCollectableAttribute.js file.

Reply:
GH was tricking me, didn't see the usage of UCATemplateValues at first